views/components/ConfigurationsList/index.jsx

```diff
@@ -568,22 +600,24 @@ const ConfigurationsList = ({ user, classes, theme }) => {
     return cards
   }
   const shareWithUsers = () => {
+    const { _id } = state.selectedConfig
+    const configAttributes = {
+      readers: state.shared.map((sharedWith) => {
+        return sharedWith.value
+      }),
```

Comment:
🎱 you can one-line this if you want

```suggestion
      readers: state.shared.map((sharedWith) => sharedWith.value),
```

Reply:
[2a243ef](https://github.com/AMP-SCZ/dpdash/pull/305/commits/2a243ef36a4f2e61ac2f5a1fccb5c52868e18464)